api/autoload/autoload.php

```diff
@@ -0,0 +1,15 @@
+<?php
+class Autoloader
+{
+    public static function loader($class)
+    {
+        $filename ='api/'. $class . '.php';
```

Comment:
Generally, you should avoid hard-coding directory names inside the string('api'). In this case Autoloader will only have one place to look for. Better approach would be to define a list of directories inside an array and look in every one of them if the file exists.